src/components/auth/auth-ui.js

```
@@ -0,0 +1,76 @@
+      logoutBtn.disabled = true;
+      userInfoDiv.textContent = 'Logged out';
+    }
+  });
+
+  listeners.push(unsubscribe);
+
+  const cleanup = () => {
+    cleanupListeners();
+    isInitialized = false;
+    parentElement && parentElement.removeChild(authContainer);
+    authContainer = null;
```

Biome (2.1.2)

[error] Can't assign authContainer because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)